index.js

```diff
@@ -127,25 +144,27 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
-
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    let userInventory = message.author.id;
```

Comment:
if something doesn't change, we should use `const`. Almost everything will be a `const`, so having a `let` is an active signal that this variable is going to change.

Also, the name for this variable implies that it holds the entire inventory of a user. In actuality it's the id of user, so we can just call this something like `const userId`